docs/manage/configuration.md

```diff
@@ -0,0 +1,283 @@
+  <file>/etc/viam.json</file> is just the default location.
+
+## The CONFIG tab
+
+The **CONFIG** tab is a central feature of your robot's page on the [Viam app](https://app.viam.com).
+
+You can use the mode selector to switch between **Builder** and **Raw JSON**:
+
+- **Builder** mode provides a graphical interface for configuring your robot resources.
+It generates properly formatted JSON for you.
+- **Raw JSON** mode provides a text editing field where you can write and edit the full config manually.
+It can be convenient to view a large config in raw JSON mode because all the components, services, modules and more that you configure in individual tabs of builder mode will appear in one long file in raw JSON mode.
+You can also copy and paste the raw JSON if needed.
+
+![The CONFIG tab of the Viam app with a box highlighting the CONFIG tab and the Mode selector (where you can select Builder or Raw JSON).](../img/configuration/config-tab.png)
+
+If you add components in **Builder** mode and click **Save Config** at the bottom of the screen, you can switch to **Raw JSON** and see the JSON that has been generated by the builder.
+
+The sections of the robot config (which each have corresponding sub-tabs on the **CONFIG** tab) are as follows:
+
+## Components
+
+Components represent the pieces of hardware on your robot that you want to control with Viam.
+Find information on how to configure each supported component type in its respective [documentation](/components/).
+You must configure each component with a name, a model, a type, attributes, and dependencies:
+
+- `name`: Serves as an identifier when accessing the resource from your code, as well as when configuring other resources that are dependent on that resource.
+You can choose any unique name for a component.
+
+- `type`: The broad component category, such as `motor`, `arm` or `camera`.
+  Components of a given type have a common API.
+
+- `model`: Indicates the more specific category of hardware.
+Components of the same model are supported using the same low-level code.
+
+- `attributes`: A struct to define things like how the component is wired to the robot, its dimensions, and other specifications; attributes vary widely between models.
+  See the [component documentation](/components/) for a given component type and model for more details.
+
+- `depends_on`: Any components that a given component relies upon, and that must be initialized on boot before this component is initialized.
+  Many built-in components have convenient implicit dependencies, in which case `depends_on` can be left blank.
+  For example, a [`gpio` motor](/components/motor/gpio/) depends on the `board` to which it is wired, but it has a dedicated `board` attribute and `viam-server` will automatically initialize that board before it looks for the motor.
+
+When you configure a component on the **CONFIG** tab, it will also appear on the **CONTROL** tab which gives you an interface to test and interact with it.

```

Comment:
Ooh I would but i just moved a different sentence to above this and want to call *it* out as a tip too so then it will look weird. hmm

Reply:
maybe put the enitre segment into it
```suggestion
{{% alert title="Tip" color="tip" %}}
Find specific information on how to configure each supported component type in its respective [documentation](/components/).

When you configure a component on the **CONFIG** tab, it will also appear on the **CONTROL** tab which gives you an interface to test and interact with it.
Meanwhile the **CODE SAMPLE** tab will also update to include code for some basic interaction with that component using the Viam [SDKs](/program/sdk-as-client/).
{{% /alert %}}
```

Reply:
Mmm but those are two very different concepts. Maybe we should just have two back to back alerts? Would that be weird?

Reply:
Yeah. I think it's fine to have with the space in between

Reply:
Done!